docs/app/Examples/modules/Popup/Types/PopupHtmlExample.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Popup, Card, Rating, Image } from 'stardust'
+
+const users = [
+  {
+    name: 'Elliot Fu',
+    bio: 'Elliot has been a member since July 2012',
+    avatar: 'http://semantic-ui.com/images/avatar/small/elliot.jpg',
+  },
+  {
+    name: 'Stevie Feliciano',
+    bio: 'Stevie has been a member since August 2013',
+    avatar: 'http://semantic-ui.com/images/avatar/small/stevie.jpg',
+  },
+  {
+    name: 'Matt',
+    bio: 'Matt has been a member since July 2014',
+    avatar: 'http://semantic-ui.com/images/avatar/small/matt.jpg',
+  },
+]
+
+export default function () {
+  const IndividualCard = (
+    <Card>
+      <Image src='http://semantic-ui.com/images/movies/totoro-horizontal.jpg' />
+      <Card.Content>
+        <Card.Header>
+          My Neighbor Totoro
+        </Card.Header>
+        <Card.Description>
+          Two sisters move to the country with their father in order to be
+          closer to their hospitalized mother, and discover the surrounding
+          trees are inhabited by magical spirits.
+        </Card.Description>
+      </Card.Content>
+    </Card>
+  )
+
+  const content = <div>
+    <div className='header'>User Rating</div>
+    <div className='content'>
+      <Rating icon='star' defaultRating={3} maxRating={4} />
+    </div>
+  </div>
+
+  return <Popup
+    trigger={IndividualCard}
+    content={content}
+    hover
+  />
```

Comment:
Few points on this one:

1. Let's use `children` for defining full markup
1. Return multi-line JSX inside of parens (`npm run lint` to see all issues)
1. Use sub components for SUI component parts. See [this section](https://github.com/TechnologyAdvice/stardust/blob/master/.github/CONTRIBUTING.md#sui-html-markup) in the CONTRIBUTING.md for more details on API design.

Given our design and style patterns noted here ^, this example would look more like this:

```jsx
return (
  <Popup trigger={IndividualCard} hover>
    <Popup.Header>User Rating</Popup.Header>
    <Popup.Content>
      <Rating icon='star' defaultRating={3} maxRating={4} />
    </Popup.Content>
  </Popup>
)
```

Note, I also realize now that there is a name conflict between the `data-title` prop and the `class="header"` component part.  Let's go with the Popup class names (`header`, etc) for our prop names and sub component names.